### DISCUSSION TITLE
Have you made your own theme?

### DISCUSSION THREAD

**Discussion by Synaptic149** [t1m0thyj/WinDynamicDesktop#438] [Polls]
If you have made a custom theme, what was your experience?

**Comment by ilverogattuso** [comment]
I have made some custom themes using the JSON method, and fortunately the setup part it's not that hard as I thought it would, I'd say I have struggled more with finding wallpapers sources haha, for the time being I've been using some of Arsenixc illustrations on Deviantart (ofc for personal use only) and Sony Global (https://www.sony.net/united/clock/), but I'd like to know if there are more sources I could use, so if you happen to know any, I'd love to hear them! Thanks.

**Comment by pacman2108** [comment]
I made it by manually editing a ddw file using 7-Zip

**Comment by Sereoj** [comment]
I am creating a theme using the site wallone.ru
